packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -241,6 +241,60 @@ void main() {
       expect(findStatic(), findsOneWidget);
     });
 
+    testWidgets('_DecoyChild preserves the child color', (WidgetTester tester) async {
+      final Widget child = getChild();
+      await tester.pumpWidget(CupertinoApp(
+        home: CupertinoPageScaffold(
```

Comment:
The indentation should be 2 spaces here in this pumpWidget.

Reply:
Fixed in 7e80b4f52ec920b301fec11d8ea2371facecec1c